src/mhw_lookup.cpp

```diff
@@ -110,12 +126,39 @@ namespace {
 		if(!std::regex_match(realid, IncludeMonsterIdRegex))
 			return false;
 		m.used = true;
+
+		m.hp_total = mb.read_mem<float>(hcompaddr + offsets::MonsterHealthComponent::MaxHealth, true);
+		m.hp_current = mb.read_mem<float>(hcompaddr + offsets::MonsterHealthComponent::CurrentHealth, true);
+		float size_scale = mb.read_mem<float>(maddr + offsets::Monster::MonsterSizeScale, true);
```

Comment:
Use `const` correctness if applicable

Reply:
I applied `const` to `size_scale` but I couldn't apply it to `scale_modifier` because the latter is assigned in the next line.
```cpp
const auto size_scale = mb.read_mem<float>(maddr + offsets::Monster::MonsterSizeScale, true);
auto scale_modifier = mb.read_mem<float>(maddr + offsets::Monster::MonsterScaleModifier, true);
if (scale_modifier <= 0 || scale_modifier >= 2 ) scale_modifier = 1;
```

Reply:
That's cool - indeed was expecting this.